dashr/chapters/dashDataTable/part1/index.R

```diff
@@ -1,59 +1,315 @@
-> If you'd like to follow along on your own machine, 
+> The set of examples on this page are rendered with a few different dataframes that have different sizes
+> and shapes. In particular,
+> some of the dataframes have a large number of columns or have cells with long contents.
+> If you'd like to follow along on your own machine,
 > then open up the menu below to copy and paste the code behind these datasets.
                 "),
     
-    htmlBr(),
-
+    htmlHr(),
+    
     dccMarkdown("
-The default styles work well for a small number of columns and short text. 
-However, if you are rendering a large number of columns or cells with long contents, 
+The default styles work well for a small number of columns and short text.
+However, if you are rendering a large number of columns or cells with long contents,
 then you'll need to employ one of the following \"overflow strategies\" to keep the table within its container.
 
-> Heads up! In the future, 
-> we may modify our default styles to better accommodate wide content 
-> while keeping the table full-width and responsive. Subscribe to 
+> Heads up! In the future,
+> we may modify our default styles to better accommodate wide content
+> while keeping the table full-width and responsive. Subscribe to
 > [plotly/dash-table#197](https://github.com/plotly/dash-table/issues/197) for more.
 
-### Overflow Strategies - Multiple Lines
+View Datasets
+                "), 
+    
+    dccMarkdown("
+```
+df_election <- data.frame(
+  Date = c('July 12th, 2013 - July 25th, 2013',
+           'July 12th, 2013 - August 25th, 2013',
+           'July 12th, 2014 - August 25th, 2014'),
+  Election_Polling_Organization = c('The New York Times',
+                                    'Pew Research',
+                                    'The Washington Post'),
+  Rep = c(1, -20, 3.512),
+  Dem = c(10,20,30),
+  Ind = c(2, 10924, 3912),
+  Region = c('Northern New York State to the Southern Appalachian Mountains',
+             'Canada',
+             'Southern Vermont')
+)
+```
+              "),
+  dccMarkdown("
+## Overflow Strategies - Multiple Lines
 
-If your cells contain contain text with spaces, 
+If your cells contain contain text with spaces,
 then you can overflow your content into multiple lines.
 
-> We find that this interface is a little too complex. 
+> We find that this interface is a little too complex.
 > We're looking at simplifying this in this issue: https://github.com/plotly/dash-table/issues/188
+                "),
+  examples$multipleLinesPseudo$source_code,
+  examples$multipleLines$layout,
+  
+  dccMarkdown("
+## Overflow Strategies - Overflowing Into Ellipses
+
+Alternatively, you can keep the content on a single line but display a set of
+ellipses if the content is too long to fit into the cell.
+
+Here, `max-width` is set to 0.
+It could be any number, the only important thing is that it is supplied.
+The behaviour will be the same whether it is 0 or 50.
```

Comment:
```suggestion
The value that is supplied will not affect the rendering of the table.
```

Side note -- why is there a `max-width` if changing its value doesn't change any behaviour?

Reply:
It is in https://dash.plot.ly/datatable/sizing "Overflow Strategies - Overflowing Into Ellipses". I noticed the new version of `dashDataTable`, `max-width` **do** affect the rendering of table. However, the old version does not. So it should be erased.